src/video/sdl_v.cpp

```diff
@@ -656,55 +657,27 @@ void VideoDriver_SDL::Stop()
 	}
 }
 
-void VideoDriver_SDL::MainLoop()
```

Comment:
This code shouldn't be moved around while modifying it - difficult to tell what's been changed - should be split into separate commits